client/python/pyproject.toml

```diff
@@ -18,31 +18,30 @@
 #
 
 [tool.poetry]
-name = "polaris"
+name = "polaris.catalog"
```

Comment:
These looks like regressions to me -- do we need to fix whatever this is generated from?